start/routes.ts

```diff
@@ -21,19 +21,32 @@
 import Route from '@ioc:Adonis/Core/Route';
 
 Route.get('/', async () => {
-  return { hello: 'Diwala' };
+    return { hello: 'Diwala' };
 });
 
 Route.group(() => {
-
     //User Authentication
     Route.post('/register', 'UsersController.register');
     Route.post('/login', 'UsersController.login');
 
     //All routes in here need a bearer-token otherwise there will be unauthorized access
     Route.group(() => {
+
         Route.post('/logout', 'UsersController.logout');
-        
-    }).middleware('auth:api');
 
+        //Products
+        Route.get('/products', 'ProductsController.index');
+        Route.post('/products', 'ProductsController.store');
+        Route.get('/products/:id', 'ProductsController.show');
+        Route.patch('/products/:id', 'ProductsController.update');
+        Route.delete('/products/:id', 'ProductsController.destroy');
+
+        //Orders
+        Route.get('/orders', 'OrdersController.index');
+        Route.post('/orders', 'OrdersController.store');
+        Route.get('/orders/:id', 'OrdersController.show');
```

Comment:
Will it be so that I can see all the orders on this? Or I can see your orders?

Or another pizza store can see other pizzastores orders?

Reply:
You will see orders based on the product, however, in a more structured application, each merchant will have to ability to see only orders made based on the merchant store (merchant ID) and only the admin can view all orders for all the merchant stores.

Reply:
Sounds good